buildenv/docker/jdk9/armhf_CC/Dockerfile

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) 2017, 2017 IBM Corp. and others
```

Comment:
To match the other platforms, this Dockerfile should be in a ubuntu16 subdirectory.

Reply:
Will do.

Reply:
Thinking about this on the way to work this morning, I'm not sure ubuntu16 is correct. For other platforms ubuntu16 is the OS where the binary is built and where it runs. Since arm is cross-compiled, perhaps arm-linux-gnueabihf is more appropriate.